CHANGES.md

```diff
@@ -1,3 +1,19 @@
+## edge-19.4.1
+
+* CLI
+  * Introduced an `upgrade` command! This allows an existing Linkerd Control Plane to be reinstalled or reconfigured
+  * The `--proxy-auto-inject` flag  from the `install` command has been completely revamped
+  * The `inject` command proxy options are now converted into config annotations; the annotations ensure that these configs are persisted in subsequent resource updates
+  * Always show open TCP connections in the CLI
+  * The `stat` command now always shows the number of open TCP connections
+  * **Breaking Change** Removed the `--disable-external-profiles` flag from the `install` command; external profiles are now disabled by default and can be enabled with the new `--enable-external-profiles` flag
```

Comment:
sorry should have called this out in the initial review: recommend bumping this bullet to the beginning for emphasis, similar to the previous releases.

Reply:
No problem! @olix0r recommend moving this to the bottom originally, but I can change to whatever we prefer. I can see how we would want to highlight the more significant changes, but I agree that we in the past have began this section highlighting break changes.

Reply:
Yeah, I don't think this is a pretty unimportant item. Much less impactful than the upgrade change, etc.